source/draw/gpu/opengl/backend/EGLNativeFence.ooc

```diff
@@ -0,0 +1,28 @@
+/* This file is part of magic-sdk, an sdk for the open source programming language magic.
+ *
+ * Copyright (C) 2016 magic-lang
+ *
+ * This software may be modified and distributed under the terms
+ * of the MIT license.  See the LICENSE file for details.
+ */
+
+import egl/egl
+import GLExtensions, GLFence
+version(!gpuOff) {
+EGLNativeFence: class extends GLFence {
+	_display: Pointer
```

Comment:
This should be of type EGLDisplay, but that should be all over the code i guess so does not belong to this PR.